### PR TITLE
Add a check for compressed files to FastqProcessing task

### PR DESCRIPTION
**Pull request by jessicaway**
Add a check to rename files if they are compressed. 
Change file names in the the v3 test to test the new code.

### CODE REVIEW

File: tasks/skylab/FastqProcessing.wdl [hunk 1]
```diff
@@ -10,7 +10,8 @@ task FastqProcessing {
     String sample_id
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
+    String docker = "quay.io/humancellatlas/fastq-process:latest"
```
Comment (jessicaway) [inline tasks/skylab/FastqProcessing.wdl:13]: Why did we switch from the sctools docker? Is the dockerfile to build this image checked in somewhere?

File: tests/skylab/optimus/prV3/test_inputs.json [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
+      "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L002_R1_001.filtered"
   ],
   "TestOptimusPR.r2_fastq": [
       "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L001_R2_001.filtered.fastq.gz",
-      "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L002_R2_001.filtered.fastq.gz"
+      "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L002_R2_001.filtered.fastq.in_gzip_format"
   ],
   "TestOptimusPR.i1_fastq": [
-      "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L001_I1_001.filtered.fastq.gz",
+      "gs://hca-dcp-sc-pipelines-test-data/smallDatasets/chemistry_10X_V3/pbmc_10k_v3_chr21_genic/pbmc_10k_v3_S1_L001_I1_001.filtered.fastq",
```
Comment (jessicaway) [inline tests/skylab/optimus/prV3/test_inputs.json:16]: are any of these files uncompressed?

**Review by jessicaway — COMMENTED** [review]
Just a couple minor things

File: tests/skylab/optimus/prV3/test_inputs.json (hunk 2 above)
Comment (kishorikonwar) [reply tests/skylab/optimus/prV3/test_inputs.json:17]: Yes,  one of the R2 are named .fastq.gz but it is actually a plain .fastq

File: tasks/skylab/FastqProcessing.wdl (hunk 1 above)
Comment (kishorikonwar) [reply tasks/skylab/FastqProcessing.wdl:13]: We need to have the review done before we add the new docker. I invited you as a reviewer for that PR. All changes are cosmetic (more information is printed about the inputs)

**Review by kishorikonwar — APPROVED** [review]

**Review by khajoue2 — APPROVED** [review]
lgtm

**Pull request merged**